teps/0060-remote-resource-resolution.md

```diff
@@ -0,0 +1,386 @@
+  ever again.
+- Provide the ability to manage Tekton resources in an org's source of choice
+  via any process of their choice - `git push`, `tkn bundle push`, `gsutil cp`
+  and so on, instead of `kubectl apply`.
+- Allow multiple remotes to be supported in a single pipeline: task A from OCI,
+  task B from the Catalog, task C directly from the cluster, and so on.
+- Allow the open source Pipelines project (and any downstream project) to
+  choose which remotes are enabled out of the box with a new release.
+- Allow operators to add new remote sources without having to redeploy the
+  entirety of Tekton Pipelines.
+- Allow operators to completely remove the code paths that fetch Tekton
+  resources from remotes they don't want to support.
+- Allow operators to control the threshold at which fetching a remote Tekton
+  resource is considered timed out.
+- Emit Events and Conditions operators can use to assess whether fetching
+  remote resources is slow or failing.
+- Establish a common syntax that tool and platform creators can use to record,
+  as part of a pipeline or run, the remote location that Tekton resources
+  should be fetched from.
+- Offer a mechanism to verify remote tasks and pipelines against a digest
+  (or similar mechanism) before they are processed as resources by Pipelines'
+  reconcilers to ensure that the resource returned by a given Remote matches the
+  resource expected by the user.
+
+### Non-Goals
+
+### Use Cases (optional)
+
+**Sharing a git repo of tasks**: An organization decides to manage their
+internal tasks and pipelines in a shared git repository. The CI/CD cluster is
+set up to allow these resources to be referenced directly by TaskRuns and
+PipelineRuns in the cluster.
+
+**Using non-git VCS for pipelines-as-code**: A game dev keeps all of their
+Tasks and Pipelines, config-as-code style, in a Perforce repo alongside the
+assets and source that make up their product.  Pipelines does not provide
+support for resolving Tekton resources from Perforce repositories
+out-of-the-box. The dev creates a new Resolver by following a template and docs
+published as part of the Tekton Pipelines project. The dev's Resolver is able
+to fetch Tekton resources from their Perforce repo and return them to the
+Pipelines reconciler.
+
+**Using a cloud bucket to store tasks and pipelines**: A team keeps all of
+their CI/CD workflows in a cloud bucket so they can quickly add new features.
+They want to be able to refer to the tasks and pipelines stored in that bucket
+rather than `kubectl apply` when a new workflow is added.
+
+**Platform interoperability**: A platform builder accepts workflows imported
+from other Tekton-compliant platforms. When a Pipeline is imported that
+includes a reference to a Task in a Remote that the platform doesn't already
+have it goes out and fetches that Task, importing it alongside the pipeline
+without the user having to explicitly provide it.
```

Comment:
The other side of this coin is "what if one or all the resolver used is/are not enable on the instance" ?